src/views/search/parts/attachment-type-email-status-row.tsx

```diff
@@ -119,11 +138,18 @@ export const AttachmentTypeEmailStatusRow = ({
 							options={emailStatusOptions}
 							background="gray5"
 							disableOptions={false}
-							onAdd={emailStatusChipOnAdd}
-							onChange={onChange}
+							onAdd={(label) => {
+								if (typeof label !== 'string') {
+									return undefined;
+								}
+								return emailStatusChipOnAdd(label, value) as any;
```

Comment:
it's a pity to have to cast as any, but i understand it is a current type definition limitation on the DS. maybe we could indicate it in a comment?